integration/build.gradle

```diff
@@ -31,7 +31,7 @@ repositories {
 group = "io.opencensus"
 version = "0.13.0-SNAPSHOT" // CURRENT_OPENCENSUS_VERSION
 
-def opencensusVersion = "0.12.2" // LATEST_OPENCENSUS_RELEASE_VERSION
+def opencensusVersion = "0.13.0-SNAPSHOT" // LATEST_OPENCENSUS_VERSION we want to test.
```

Comment:
A defect is found and fixed in https://github.com/census-instrumentation/opencensus-java/pull/1072. So here I change the interop test to run against latest opencensus-java.

To run local test, one needs to run `./gradlew install` to install opencensus-java 0.13.0-SNAPSHOT first.